__tests__/test-mergeFile.js

```diff
@@ -29,33 +29,4 @@ describe('mergeFile', () => {
     expect(cleanMerge).toBe(false)
     expect(mergedText).toEqual(await fs.read(`${dir}/aoc.txt`, 'utf8'))
   })
-
-  it('mergeFile a o c --diff3', async () => {
-    // Setup
-    const { fs, dir } = await makeFixture('test-mergeFile')
-    // Test
-    const { cleanMerge, mergedText } = mergeFile({
-      ourContent: await fs.read(`${dir}/a.txt`, 'utf8'),
-      baseContent: await fs.read(`${dir}/o.txt`, 'utf8'),
-      theirContent: await fs.read(`${dir}/c.txt`, 'utf8'),
-      format: 'diff3',
-    })
-    expect(cleanMerge).toBe(false)
-    expect(mergedText).toEqual(await fs.read(`${dir}/aoc3.txt`, 'utf8'))
-  })
-
-  it('mergeFile a o c --diff3 --marker-size=10', async () => {
-    // Setup
-    const { fs, dir } = await makeFixture('test-mergeFile')
-    // Test
-    const { cleanMerge, mergedText } = mergeFile({
-      ourContent: await fs.read(`${dir}/a.txt`, 'utf8'),
-      baseContent: await fs.read(`${dir}/o.txt`, 'utf8'),
-      theirContent: await fs.read(`${dir}/c.txt`, 'utf8'),
-      format: 'diff3',
-      markerSize: 10,
-    })
-    expect(cleanMerge).toBe(false)
-    expect(mergedText).toEqual(await fs.read(`${dir}/aoc3-m10.txt`, 'utf8'))
-  })
```

Comment:
If you removed those two tests does it mean that this API no longer works? This would be a breaking change.

Reply:
Options for diff3 and custom marker size were only exposed internally, and never used by anything except these tests, so this wouldn't be a breaking change.